README.md

```diff
@@ -42,31 +42,70 @@ jobs:
           token: ${{ secrets.BUF_TOKEN }}
 ```
 
-See [action.yml](action.yml) for all options.
-
 ### Default behavior
 
-The default behavior of this action is the recommended workflow for a GitHub repository that contains Protobuf files.
+When you push a commit, tag, or branch to GitHub, the action will [push named modules to the BSR](https://buf.build/docs/bsr/module/publish).
+On a pull request, the action will run all checks and post a [summary comment](#summary-comment) on the PR.
+When you delete a branch or tag, the action will archive the corresponding label on the BSR.
+This behavior can be customized by setting the action inputs.
 
-| GitHub action event | Default behavior | `buf` commands |
-| - | - | - |
-| [`push`][push-event] | Modules that are configured with a BSR name are [pushed to the BSR](https://buf.build/docs/bsr/module/publish) every time a new Git commit, tag, or branch is pushed to GitHub. | `buf push` |
-| [`pull_request`][pull-request-event] | Run all checks and post (or update) a [summary comment](#summary-comment) on the PR every time the PR is updated. Errors are added as annotations on the PR. | `buf build`<br>`buf lint`<br>`buf format`<br>`buf breaking` |
-| [`delete`][delete-event] | Archive the corresponding label on the BSR every time a Git branch or tag is deleted from GitHub. | `buf beta registry archive --label` |
+### Configuration
 
-### Skipping steps
+To customize the behavior of the action, you can set the following inputs in the workflow file.
+Add these inputs under the `with` section of the `uses` step in the workflow file.
 
-The default configuration makes it possible to skip lint, formatting, or breaking change checks on a PR
-by adding a label with a (case-insensitive) special name to that PR.
+```yaml
+- uses: bufbuild/buf-action@v0.1
+  with:
+    ...
+```
 
-- `buf skip breaking`: skips breaking change detection.
-- `buf skip lint`: skips lint.
-- `buf skip format`: skips format. 
+| Parameter                       | Description                                        | Default            |
+|:--------------------------------|:---------------------------------------------------|:-------------------|
+| `archive`                       | Whether to run the archive step. | Runs on Git deletes |
```

Comment:
Looks like this is alphabetical order? IIRC we wanted to order this most important first.

Reply:
@bufdev can you remind us what you wanted here?

Reply:
Changed ordering.